dlk/python/dlk/plugins/tf.py

```diff
@@ -391,6 +391,14 @@ def convert_operator(self, op_type: str) -> str:
         dlk_op_type = DLK_OPERATOR_MAP.get(op_type)
         return dlk_op_type if dlk_op_type else op_type
 
+    def convert_deprecated_operator(self, op_type: str) -> str:
+        """Support backwards compatibility to old operators"""
+        DEPRECATED_OPERATOR_MAP: Dict[str, str] = {
```

Comment:
Thanks. Cool PR!
How about put DEPRECATED_OPERATOR_MAP to outside of this functio?

Reply:
If you created this function just for this operator, this function name is too generic and maybe good to hard write this functionality in original function..
Anyway I want @tkng san review

Reply:
@iizukak Thanks for the comments. I returned the function back to its original form `convert_operator`.